gemet/thesaurus/management/commands/importspreadsheet.py

```diff
@@ -107,6 +107,39 @@ def _create_concepts(self, sheet):
                 if search_text:
                     self.properties.append(search_text)
 
+    def _create_theme_group_relations(self, source):
+        property_types = models.PropertyType.objects.filter(
+            name__in=['theme', 'group']
+        )
+        property_type_broader = models.PropertyType.objects.get(name='broader')
+        for property_type in property_types:
+            relation = source.source_relations.filter(
+                property_type=property_type).first()
```

Comment:
There is no need to fetch a record if you don't use it. To check for existence use [exists()](https://docs.djangoproject.com/en/1.10/ref/models/querysets/#django.db.models.query.QuerySet.exists).